data/txHandler.go

```diff
@@ -171,13 +171,11 @@ func MakeTxHandler(opts TxHandlerOpts) (*TxHandler, error) {
 		streamVerifierDropped: make(chan *verify.UnverifiedTxnSigJob),
 	}
 
-	// use defaultBacklogSize = approx number of txns in a full block as a parameter for the dedup cache size
```

Comment:
Your PR describes what change is being made, but could you also include why? it would be helpful if we ever need to refer back to this change. This comment describes a reasonable estimation, so why 10x it?
(I know we've discussed it plenty in team settings, but I want to make sure it's recorded)